Dockerfile

```diff
@@ -5,9 +5,7 @@ COPY ./deviceauth /usr/bin/
 RUN mkdir /etc/deviceauth
 COPY ./config.yaml /etc/deviceauth/
 
-# example server private key - only for testing purpose
-# this key will be replaced with proper one later
+# mount your private key at /etc/deviceauth/rsa/private.pem
```

Comment:
docker-compose also has some commented out volume placeholders - we could add one there, so that's all in one place for reference (writing docs, prod scripts)